modules/nw-sriov-configure-exclude-topology-manager.adoc

```diff
@@ -0,0 +1,149 @@
+metadata:
+  name: <policy_name>
+  namespace: openshift-sriov-network-operator
+spec:
+  resourceName: sriovnuma0 <1>
+  nodeSelector:
+    kubernetes.io/hostname: <node_name>
+  numVfs: <number_of_Vfs>
+  nicSelector: <2>
+    vendor: "<vendor_ID>"
+    deviceID: "<device_ID>"
+  deviceType: netdevice
+  excludeTopology: true <3>
+----
+<1> The resource name of the SR-IOV network device plugin. This YAML uses a sample `resourceName` value.
+<2> Identify the device for the Operator to configure by using the NIC selector.
+<3> To exclude advertising the NUMA node for the SR-IOV network resource to the Topology Manager, set the value to `true`. The default value is `false`.
+
+.. Create the `SriovNetworkNodePolicy` resource by running the following command:
++
+[source,terminal]
+----
+$ oc create -f sriov-policy-numa.yaml
+----
++
+.Example output
+[source,terminal]
+----
+sriovnetworknodepolicy.sriovnetwork.openshift.io/policy-for-numa-0 created
+----
+
+. Create the `SriovNetwork` CR:
+
+.. Save the following YAML in the `sriov-network-numa.yaml` file, replacing values in the YAML to match your environment:
++
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetwork
+metadata:
+  name: sriov-numa-0-network <1>
+  namespace: openshift-sriov-network-operator
+spec:
+  resourceName: sriovnuma0 <2>
+  networkNamespace: <namespace> <3>
+  ipam: |- <4>
+    {
+      "type": "<ipm_type>", 
+    }
+----
+<1> Replace `sriov-numa-0-network` with the name for the SR-IOV network resource. 
+<2> Specify the resource name for the `SriovNetworkNodePolicy` CR from the previous step. This YAML uses a sample `resourceName` value. 
+<3> Enter the namespace for your SR-IOV network resource.
+<4> Enter the IP address management configuration for the SR-IOV network.
+
+.. Create the `SriovNetwork` resource by running the following command:
++
+[source,terminal]
+----
+$ oc create -f sriov-network-numa.yaml
+----
++
+.Example output
+[source,terminal]
+----
+sriovnetwork.sriovnetwork.openshift.io/sriov-numa-0-network created
+----
+
+. Create a pod and assign the SR-IOV network resource from the previous step:
+
+.. Save the following YAML in the `sriov-numa-pod.yaml` file, replacing values in the YAML to match your environment:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: <pod_name>
+  annotations:
+    k8s.v1.cni.cncf.io/networks: |-
+      [
+        {
+          "name": "sriov-numa-0-network", <1>
+        }
+      ]
+spec:
+  containers:
+  - name: <container_name>
+    image: <image> 
+    imagePullPolicy: IfNotPresent
+    command: ["sleep", "infinity"]
+----
+<1> This is the name of the `SriovNetwork` resource that uses the `SriovNetworkNodePolicy` resource.
+
+.. Create the `Pod` resource by running the following command:
++
+[source,terminal]
+----
+$ oc create -f sriov-numa-pod.yaml
+----
++
+.Example output
+[source,terminal]
+----
+pod/example-pod created
+----
+
+.Verification
```

Comment:
To show the pod is scheduled with SR-IOV resources in a different NUMA node than the CPU, the following commands can be executed in the pod context:

```
$ lscpu | grep NUMA
NUMA node(s):                    2
NUMA node0 CPU(s):     0,2,4,6,8,10,12,14,16,18,...
NUMA node1 CPU(s):     1,3,5,7,9,11,13,15,17,19,...

$ cat /proc/self/status | grep Cpus
Cpus_allowed:	aa
Cpus_allowed_list:	1,3,5,7

$ cat  /sys/class/net/net1/device/numa_node
0
```

1,3,5,7 CPUs belongs to NUMA node1, while the nic is in node0.

Please note that even with `exludeTopology` set to true, the assigned resources can be in the same NUMA node.